purchases/src/main/kotlin/com/revenuecat/purchases/PurchasesConfiguration.kt

```diff
@@ -94,10 +94,12 @@ open class PurchasesConfiguration(builder: Builder) {
          * The result of the verification can be obtained from [EntitlementInfos.verification] or
          * [EntitlementInfo.verification].
          *
-         * This feature is currently in beta and the behavior may change.
+         * This feature is currently in beta and the behavior may change. Only available in Kotlin.
          *
          * Default mode is disabled.
          */
+        @JvmSynthetic
```

Comment:
Are we planning on releasing this on the hybrids before it's not experimental?

Reply:
Hmm I don't think we will release it in the hybrids while it's experimental... IMO, we should wait until we are confident in the natives before porting to the hybrids. Lmk if you think otherwise though!